.husky/pre-commit

```diff
@@ -2,3 +2,4 @@
 . "$(dirname -- "$0")/_/husky.sh"
 
 npx pretty-quick --staged
+npx lint-staged
```

Comment:
Aha, yes, I used to like `pretty-quick` but (for reasons I no longer remember) ended up switching to `lint-staged`. You can go ahead and remove `pretty-quick` from the repo.